src/modules/search/searchApi.js

```diff
@@ -13,8 +13,21 @@ import {
 } from '../../util/apiHelpers';
 
 const baseUrl = apiResourceUrl('/search-api/v1/search');
+const articleUrl = apiResourceUrl('/article-api/v2/articles');
 
 export const search = (queryString, locale) =>
   fetchAuthorized(`${baseUrl}/${queryString}&language=${locale}`).then(
     resolveJsonOrRejectWithError,
   );
+
+export const searchArticles = (queryString, locale) =>
+  fetchAuthorized(`${articleUrl}/${queryString}?language=${locale}`).then(
+    resolveJsonOrRejectWithError,
+  );
+
+export const searchRelatedArticles = async (input, locale) => {
+  await new Promise(resolve => setTimeout(resolve, 50));
```

Comment:
Er det noen spesiell grunn til at det settes en timeout her?

Reply:
Siden det søkes hver gang input forandres, og søket er async, så kom ikke svarene alltid i rekkefølge. En liten timeout sørger for at rekkefølgen beholdes